docs/_data/meetups/usa-washingtondc.yaml

```diff
@@ -0,0 +1,11 @@
+region: North America
+country: USA
+state: DC
+city: Washington
+page: ./dc/
```

Comment:
This page doesn't exist (which is why the build failed). I think you can just remove this line and it should pass 👍

Reply:
Ok, I updated it. Hopefully it passes next time.